src/syntax-tree/ast.ts

```diff
@@ -1794,21 +1860,89 @@ export class FunctionCallExpr extends Expression {
     getFunctionName(): string {
         return this.functionName;
     }
+
+    getKeyword(): string {
+        return this.functionName;
+    }
+
+    validateImport(importedModule: string, importedItem: string): boolean {
```

Comment:
These functionalities are the exact same for `FunctionCallStmt` and `FunctionCallExpr`, but because `Expression` inherits from `Statement` there is no easy way to implement these functionalities in some parent class from which both the FunctionCallStmt and FunctionCallExpr could inherit them from. 

This is kind of an issue since this code is now duplicated and we have to constantly remember to update both of these places.